app/javascript/hyacinth_ui_v1/components/digital_objects/children/Children.js

```diff
@@ -1,39 +1,37 @@
 import React from 'react';
-import produce from 'immer';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/react-hooks';
 
-import digitalObjectInterface from '../digitalObjectInterface';
+import DigitalObjectInterface from '../DigitalObjectInterface';
 import TabHeading from '../../ui/tabs/TabHeading';
-
-class Children extends React.Component {
-  // state = {
-  //   digitalObject: {},
-  // }
-
-  // componentDidMount() {
-  //   const { match: { params: { id } } } = this.props;
-  //
-  //   digitalObject.get(id)
-  //     .then((res) => {
-  //       console.log(res.data);
-  //       this.setState(produce((draft) => {
-  //         draft.digitalObject = res.data.digitalObject;
-  //       }));
-  //     });
-  // }
-
-  render() {
-    const { data: { uid } } = this.props;
-
-    return (
-      <>
-        <TabHeading>Manage Child Assets</TabHeading>
-        <p>Child Digital Objects</p>
-        <p># Total</p>
-
-
-      </>
-    );
-  }
+import { getChildrenDigitalObjectQuery } from '../../../graphql/digitalObjects';
```

Comment:
Ugh - I see why you had to do this component, but - I also did this in my branch. I'll have to do some rebasing if we merge this first.